### PR TITLE
Improve view that shows details a metrics endpoint

### PR DESCRIPTION
**Pull request by nwmac**

### CODE REVIEW

File: src/frontend/packages/core/sass/_all-theme.scss [hunk 1]
```diff
@@ -65,7 +65,8 @@
 @import '../../cloud-foundry/src/features/cloud-foundry/tabs/cloud-foundry-firehose/cloud-foundry-firehose.component.theme';
 @import '../../cloud-foundry/src/features/service-catalog/service-catalog-page/service-catalog-page.component.theme';
 @import '../../cloud-foundry/src/features/applications/application-wall/application-wall.component.theme';
-@import '../../core/src/features/error-page/error-page/error-page.component.theme.scss';
+@import '../../core/src/features/error-page/error-page/error-page.component.theme';
+@import '../../core/src/features/metrics//metrics//metrics.component.theme';
```
Comment (richard-cox) [inline src/frontend/packages/core/sass/_all-theme.scss:69]: `//metrics//` This does work, but should be fixed

File: src/frontend/packages/core/src/features/metrics/metrics-endpoint-details/metrics-endpoint-details.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="metrics-details" *ngIf="data$ | async as info">
+  <div class="metrics-details__line">
+    <mat-icon *ngIf="info.warning" class="metrics-details__icon text-warning" matTooltip="Username">warning</mat-icon>
```
Comment (richard-cox) [inline src/frontend/packages/core/src/features/metrics/metrics-endpoint-details/metrics-endpoint-details.component.html:3]: Old tooptip

**Review by richard-cox — COMMENTED** [review]
- Two super minor comments
- If two metrics instances consume metrics from the same endpoint one of those metrics instance will state the endpoint can't be found. I don't know if we want to fix this (could be costly) given that the scenario is likely a mistake

**Comment by nwmac** [comment]
Brought in by upstream merge

**Pull request closed without merging**